app/components/DepositWithdraw/WithdrawModalNew.jsx

```diff
@@ -0,0 +1,834 @@
+          </div>
+
+          {
+            !isBTS && selectedAsset && !selectedGateway ?
+            <Translate content="modal.withdraw.no_gateways" /> : 
+            null
+          }
+
+          {/*GATEWAY SELECTION*/}
+          <div style={{marginBottom: "1em"}}>
+            { selectedGateway ? gatewaySelector.call(this, {
+                selectedGateway, 
+                gatewayStatus, 
+                nAvailableGateways, 
+                error: false,
+                onGatewayChanged: this.onGatewayChanged.bind(this)
+            }) : null}
+          </div>
+
+          {/*QUANTITY*/}
+          {
+            assetAndGateway || isBTS ? 
+            <div>
+              {(preferredCurrency) ? <div style={{fontSize: "0.8em", position: "absolute", right: "1.25em"}}>
+                <Translate content="modal.withdraw.available" />
+                <span style={{color: canCoverWithdrawal ? null : "red", cursor: "pointer", textDecoration: "underline"}} onClick={this.onClickAvailableBalance.bind(this, convertedBalance)}>
+                    {/*Some currencies do not appear in balances, display zero balance if not found*/}
+                    {withdrawalCurrencyBalanceId ? <BalanceComponent balance={withdrawalCurrencyBalanceId} /> : "0.00"}
+                </span>
+              </div> : null}
+              <label className="left-label">
+                <Translate content="modal.withdraw.quantity" />
+              </label>
+              <ExchangeInput value={quantity} onChange={this.onQuantityChanged.bind(this)} onFocus={onFocus} onBlur={onBlur} />
+            </div> : 
+            null
+          }
+
+          {
+            (assetAndGateway || isBTS) && !canCoverWithdrawal ? 
+            <div style={{marginBottom: "1em"}}>
+              <Translate content="modal.withdraw.cannot_cover" />
+            </div> :
+            null
+          }
+
+          {/*ESTIMATED VALUE*/}
+          {
+            (assetAndGateway || quantity) && !isBTS ? 
+            <div>
+              <label className="left-label"><Translate content="modal.withdraw.estimated_value" /> ({preferredCurrency})</label>
+              <ExchangeInput value={userEstimate != null ? userEstimate : estimatedValue} onChange={this.onEstimateChanged.bind(this)} onFocus={onFocus} onBlur={onBlur} />
+            </div> :
+            null
+          }
+
+          {/*WITHDRAW ADDRESS*/}
+          {
+            (assetAndGateway && !isBTS) ? 
+            <div style={{marginBottom: "1em"}}>
+              <label className="left-label">
+                  <Translate component="span" content="modal.withdraw.address"/>
+              </label>
+              {addressError ? <div className="has-error" style={{position: "absolute", right: "1em", marginTop: "-30px"}}>
+                  <Translate content="modal.withdraw.address_not_valid" coin_type={selectedAsset} />
```

Comment:
Missing `coin_type` in translation file 
`Warning: Unknown prop `coin_type` on <span> tag.`

Reply:
I changed the message and removed this attribute